runtime/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -315,150 +305,72 @@ static void regclass_graph_OutputTensorInfo(py::module m) {
-           const std::shared_ptr<ov::preprocess::InputTensorInfo>& inputTensorInfo) {
-            me->tensor(std::move(*inputTensorInfo));
-            return me;
-        },
-        py::arg("tensor"),
-        R"(
-                Adds builder for actual tensor information of client's input.
-                Parameters
-                ----------
-                tensor : InputTensorInfo
-                    Client's input tensor information. It's internal data will be moved to parent InputInfo object.
-                Returns
-                ----------
-                tensor : InputInfo
-                    Reference to itself to allow chaining of calls in client's code in a builder-like manner.
-              )");
-    inp.def(
-        "preprocess",
-        [](const std::shared_ptr<ov::preprocess::InputInfo>& me,
-           const std::shared_ptr<ov::preprocess::PreProcessSteps>& preProcessSteps) {
-            me->preprocess(std::move(*preProcessSteps));
-            return me;
-        },
-        py::arg("pre_process_steps"),
-        R"(
-                Adds builder for actual preprocessing steps for input parameter.
-                Steps can specify various actions, like 'mean', 'scale' and others.
-                Parameters
-                ----------
-                pre_process_steps : PreProcessSteps
-                    Preprocessing steps. It's internal data will be moved to parent InputInfo object.
-                Returns
-                ----------
-                preprocess : InputInfo
-                    Reference to itself to allow chaining of calls in client's code in a builder-like manner.
-              )");
-    inp.def(
-        "network",
-        [](const std::shared_ptr<ov::preprocess::InputInfo>& me,
-           const std::shared_ptr<ov::preprocess::InputNetworkInfo>& inputNetworkInfo) {
-            me->network(std::move(*inputNetworkInfo));
-            return me;
-        },
-        py::arg("input_network_info"));
+    inp.def("tensor", [](ov::preprocess::InputInfo& me) {
+        return &me.tensor();
+    });
+    inp.def("preprocess", [](ov::preprocess::InputInfo& me) {
+        return &me.preprocess();
+    });
+    inp.def("network", [](ov::preprocess::InputInfo& me) {
+        return &me.network();
+    });
 }
 
 static void regclass_graph_OutputInfo(py::module m) {
-    py::class_<ov::preprocess::OutputInfo, std::shared_ptr<ov::preprocess::OutputInfo>> out(m, "OutputInfo");
+    py::class_<ov::preprocess::OutputInfo, ref_wrapper<ov::preprocess::OutputInfo>> out(m, "OutputInfo");
     out.doc() = "openvino.impl.preprocess.OutputInfo wraps ov::preprocess::OutputInfo";
 
     out.def(py::init<>(), R"(Default constructor, can be used only for networks with exactly one output)");
     out.def(py::init<size_t>(), R"(Constructor with parameter index as argument)");
     out.def(py::init<const std::string&>(), R"(Constructor with tensor name as argument)");
```

Comment:
Please remove this 'init' as well - user doesn't need to create such objects (there is no place where it can be passed)

Reply:
Removed.